lib/cmd_tool_prefix_addr.go

```diff
@@ -33,6 +33,8 @@ func CmdToolPrefixAddr(f CmdToolPrefixAddrFlags, args []string, printHelp func()
 				return err
 			}
 			fmt.Printf("%s,%s\n", input, prefix.Addr())
+		default:
+			return ErrInvalidInput
```

Comment:
We shouldn't add this - we want to ignore invalid inputs, as that gives for a cleaner interface for users who have files they pipe in with extra data sometimes.